onebusaway-android/src/main/java/org/onebusaway/android/provider/ObaProvider.java

```diff
@@ -481,6 +487,8 @@ private void dropTables(SQLiteDatabase db) {
                 ObaContract.Regions.SUPPORTS_OBA_REALTIME);
         sRegionsProjectionMap.put(ObaContract.Regions.SUPPORTS_SIRI_REALTIME,
                 ObaContract.Regions.SUPPORTS_SIRI_REALTIME);
+        sRegionsProjectionMap.put(ObaContract.Regions.SUPPORTS_EMBEDDED_SOCIAL,
```

Comment:
This is the most recently added database field, so please move to the end of the list here for the regions contract.